src/tests/ftest/control/dmg_storage_query.py

```diff
@@ -57,7 +58,7 @@ def test_dmg_storage_query_devices(self):
         :avocado: tags=DmgStorageQuery,test_dmg_storage_query_devices
         """
         # Get the storage device information, parse and check devices info
-        device_info = get_storage_query_device_info(self, self.dmg)
+        device_info = get_storage_query_device_info(self.dmg)
```

Comment:
Without a `self` argument, now I wonder if we should have this :)
```suggestion
        device_info = self.dmg.get_storage_query_device_info()
```